Traceback (most recent call last):
  File "/home/ubuntu/group_1_work/technical_assistant/error_check_file_04.py", line 8, in <module>
    print(factorial(1000))
          ^^^^^^^^^^^^^^^
  File "/home/ubuntu/group_1_work/technical_assistant/error_check_file_04.py", line 7, in factorial
    return n * factorial(n-1)
               ^^^^^^^^^^^^^^
  File "/home/ubuntu/group_1_work/technical_assistant/error_check_file_04.py", line 7, in factorial
    return n * factorial(n-1)
               ^^^^^^^^^^^^^^
  File "/home/ubuntu/group_1_work/technical_assistant/error_check_file_04.py", line 7, in factorial
    return n * factorial(n-1)
               ^^^^^^^^^^^^^^
  [Previous line repeated 996 more times]
RecursionError: maximum recursion depth exceeded
